### ISSUE TITLE
Elapsed Time Bug fixed by #3498 seems to be back in v1.50.2

### ISSUE THREAD

**Issue by Chewie9999** [rclone/rclone#3860]
#### What is the problem you are having with rclone?

Elasped Time seems to not report the correct time elapsed during the checking process

#### What is your rclone version (output from `rclone version`)

```
rclone v1.50.2
- os/arch: windows/amd64
- go version: go1.13.4
```

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)

Windows Server 2016

####  Which cloud storage system are you using? (eg Google Drive)

The Box

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
```
rclone --config rclone.conf --exclude-from exclude.rclone -P copy "remote:Property Infrastructure & Server Engineering/Server Deployment Repository/DeploymentShare$" "Y:\\ServerDeploymentShare\\DeploymentShare$"
```
#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)

Please note that the following was manually typed as I if I try and select the text in the command prompt, it doesn't copy anything, so the spacing might be slightly off:
```
------------- sync Server Box Repository ------------------------------
Transferred:               0 / 0 Bytes, -, 0 Bytes/s, ETA -
Errors:                      0
Checks:                    69475 / 69475, 100%
Transferred:               0 / 0, -
Elapsed time:             200ms
```
This scan has actually taken ~15minutes to get to this point, not 200ms

It looks like this problem was fixed in #3498 , but in the latest version is back again.

**Comment by ncw** [comment]
Hmm, this is in the check phase...

What `Elapsed time` means now is time that files are copying for and since you haven't copied any files yet that is why it is nearly zero.  We made it like that so we could compute average transfer rates properly.

However I don't think it is what users expect for something called Elapsed time...  So maybe we need two time counters one elapsed time and the other transfer time or something like that.

We probably don't need to show transfer time to users normally.

What do you think?

**Comment by Chewie9999** [comment]
Yes,in my opinion it would be good to separate times, as it's good to know how long it takes to check a source, even though there's no downloads.
I know this could be wrapped in another command to get total run time, but still...
I guess if you don't want to change the functionality, it would be good to rename it to something more specific (like Transfer Time as you suggest).
